modules/deploy/partials/cloud/private-links-test-connection.adoc

```diff
@@ -0,0 +1,64 @@
+. Set the following environment variables.
++
+[,bash]
+----
+export REDPANDA_BROKERS='<kafka-api-bootstrap-server-hostname>:30292'
+export RPK_TLS_ENABLED=true
+export RPK_USER=<user>
+export RPK_PASS=<password>
+----
+
+. Create a test topic.
++
+[,bash]
+----
+rpk topic create test-topic
+----
+
+. Produce to the test topic.
++
+[tabs]
+====
+rpk::
++
+[,bash]
+----
+echo 'hello world' | rpk topic produce test-topic
+----
+Curl::
++
+[,bash]
+----
+curl -s \
+  -X POST \
+  "<http-proxy-bootstrap-server-url>/topics/test-topic" \
+  -H "Content-Type: application/vnd.kafka.json.v2+json" \
+  -d '{
+  "records":[
+      {
+          "value":"hello world"
+      }
+  ]
+}'
+----
+====
+
+. Consume from the test topic.
++
+[tabs]
+====
+rpk::
++
+[,bash]
+----
+rpk topic consume test-topic -n 1
+----
+Curl::
```

Comment:
I think the tool is written curl (lowercase) in the official docs. We should do a find/replace to make sure our docs are consistent: https://curl.se/docs/